ui/components/providers/radio-group-provider.tsx

```diff
@@ -5,24 +5,82 @@ import React from "react";
 import { Control, Controller } from "react-hook-form";
 import { z } from "zod";
 
-import { addProviderFormSchema } from "@/types";
-
 import {
   AWSProviderBadge,
   AzureProviderBadge,
   GCPProviderBadge,
+  GitHubProviderBadge,
   KS8ProviderBadge,
   M365ProviderBadge,
-} from "../icons/providers-badge";
-import { CustomRadio } from "../ui/custom";
-import { FormMessage } from "../ui/form";
+} from "@/components/icons/providers-badge";
+import { CustomRadio } from "@/components/ui/custom";
+import { FormMessage } from "@/components/ui/form";
+import { addProviderFormSchema, IconSvgProps } from "@/types";
 
 interface RadioGroupProviderProps {
   control: Control<z.infer<typeof addProviderFormSchema>>;
   isInvalid: boolean;
   errorMessage?: string;
 }
 
+const PROVIDERS_CONFIG = [
+  {
+    value: "aws",
+    name: "Amazon Web Services",
+    description: "Amazon Web Services",
+    BadgeComponent: AWSProviderBadge,
+  },
+  {
+    value: "gcp",
+    name: "Google Cloud Platform",
+    description: "Google Cloud Platform",
+    BadgeComponent: GCPProviderBadge,
+  },
+  {
+    value: "azure",
+    name: "Microsoft Azure",
+    description: "Microsoft Azure",
+    BadgeComponent: AzureProviderBadge,
+  },
+  {
+    value: "m365",
+    name: "Microsoft 365",
+    description: "Microsoft 365",
+    BadgeComponent: M365ProviderBadge,
+  },
+  {
+    value: "kubernetes",
+    name: "Kubernetes",
+    description: "Kubernetes",
+    BadgeComponent: KS8ProviderBadge,
+  },
+  {
+    value: "github",
+    name: "GitHub",
+    description: "GitHub",
+    BadgeComponent: GitHubProviderBadge,
+  },
+] as const;
+
+const ProviderRadio = ({
```

Comment:
new component? is this needed?

Reply:
Can you answer this @alejandrobailo, please? I think you added that part.